README.md

```diff
@@ -62,15 +62,26 @@ Make sure to store and protect these values, the secret access key cannot be ret
   - Copy the `Space ID` and the `Content Delivery API - access token`, you will need those later to connect the NextJS app to Contentful
 - Enter your data into Contentful
 
-### Have a valid domain name
+### (Optional) Have a valid domain name
 
-You need to have a certified domain name to be able to deploy the app.
+If you want the application deployed with a custom domain name, you should have a certified domain name before the next steps.
+
+If you don't have a domain name, please ignore the following variables in the guide:
```

Comment:
It might be good to add a reminder before these variables down below since there's a couple of sections between this and the variables

Reply:
added an optional in front of the variables